modules/ROOT/partials/configuration/premium-upgrade-promotion-option.adoc

```diff
@@ -13,7 +13,7 @@
 
 See xref:editor-premium-upgrade-promotion.adoc#premium-upgrade-promotion-defaults[Premium upgrade promotion defaults] for details.
 
-== Example: Using `+promotion+`
+=== Example: Using `+promotion+`
```

Comment:
```suggestion
==== Example: Using `+promotion+`
```
This should be indented one more time if we've indented the rest.

Reply:
Fixed with the leveloffset